examples/spec/src/components/_pages/PageScreenHelmet.tsx

```diff
@@ -30,6 +32,7 @@ const PageScreenHelmet: React.FC = () => {
           appendLeft={appendLeft}
           appendRight={<div onClick={onRightClick}>{appendRight}</div>}
           noBorder={noBorder}
+          hideLeftButton={shouldHideLeftButton}
```

Comment:
```suggestion
          noBackButton={shouldHideLeftButton}
```

noBackButton 어때요?

- 위에 `noBorder`랑 맞춰주기
- `appendLeft`는 그대로 작동할텐데... `Left`를 가린다는게 모호함

Reply:
> noBackButton 어때요?

`noBackButton` 을 true 로 했을 때 close button 도 나타나지 않는 점이 마음에 걸려요.

> appendLeft는 그대로 작동할텐데... Left를 가린다는게 모호함

이런 측면에서 `hideLeftButton` 도 별로네요

Reply:
@tonyfromundefined 

슬랙 논의에서도 토니는 `hideBackButton` 을 제안했는데, `back` 으로 충분히 설명 가능하고 `close` 까지 커버해도 괜찮다고 생각하는 것 같아요.

혹시 이 부분에 대해 좀 더 토니의 생각을 들어볼 수 있을까요?

Reply:
아 이게 CLOSE까지 커버하는군요;; 저는 "백버튼"만 생각했어요. close 버튼은 Navigator에서 onClose 핸들러를 안 주면 숨겨지니까요.

Reply:
그럼 좀 verbose해도

`noBackAndCloseButton`은 어때요?

Reply:
> close 버튼은 Navigator에서 onClose 핸들러를 안 주면 숨겨지니까요.

아 맞아요 테스트 코드 작성할 때 이거 때문에 많이 당황했어요. 

제 경험에선 암묵적인 인터페이스라 생각했는데,

사용자 입장에서는 문서에서만 확실하게 명시해주면 `당연한` 인터페이스인지 궁금해지네요.

x 버튼에 default event 가 당연히 있을 것이라고 생각하는게 자연스러운지

x 버튼은 항상 명시적으로 event 를 선언하는 게 자연스러운지 고민이 드네요

Reply:
onClose를 안주면 숨겨진다기보다는 호출될 핸들러가 없는데 왜 X 버튼을 노출해야될까? -> 숨기자!

가 되어버린듯요... 근데 암묵적이라는건 동의해서, 차라리 onClose 핸들러를 required 필드로 만들어버리고, noClose가 ScreenHelmet에 있는게 더 나은거같아요.

Reply:
이거 back button 만 옵션 사용하도록 변경할게요. 다음과 같이 생각하고 낸 결론이에요.

- close 버튼을 눌렀을 때 default event 가 발생한다면 onClose 와 close button 을 보여주는 flag 변수는 분리하는 것이 맞다.
- 현재 close 버튼을 클릭했을 때 default event 는 발생하지 않는다. 가장 먼저 떠올릴 수 있는 default operation 은 창 닫기이지만 karrotframe 이 window.close 를 호출하는 것은 이상하다.
- 따라서 default event 를 적절하게 지원하지 않는 현재, 특별히 다루려는 이벤트가 없다면 close 버튼 역시 필요가 없고, 따라서 onClose 으로 버튼이 보여지는 것까지 관리하는 것이 타당하다.

Reply:
앗 동시에 답변을 달았군요

Reply:
토니의 솔루션은 default event 를 지원하지 않아도 event handler 와 flag 는 분리하는 것이 명시적인 코드이므로 의미가 있다는 접근이네요.

required 필드일 때 저는 종종 `const noop = () => {}` 같은 거 만들어서 넣어줬는데, 이 정도 불편함은 괜찮을 거 같아요. 따라서 찬성해요~

Reply:
근데 default event가 뭐에요?? 이해못함 ㅋㅋㅋ onClose를 말하시는거에요?

Reply:
> 근데 default event가 뭐에요?? 이해못함 ㅋㅋㅋ onClose를 말하시는거에요?

네 맞아요~

> "호출될 핸들러가 없는데 왜 X 버튼을 노출해야될까?" -> "숨기자!" 가 되어버린듯해요 ㅋㅋ

저도 이 생각에 동의했는데,

그러려면 x 버튼에 기본적으로 작동하는 핸들러가 없다는 것을 의미하니까요.

`기본적으로 작동하는 핸들러` -> default event 로 표현했어요. default handler 가 의미하는 바가 더 정확하네요.